docs/software/devtools.md

```diff
@@ -55,6 +55,37 @@ $ pre-commit run --all-files
 If you have any trouble with this tool, feel free to check out
 [the pre-commit website](https://pre-commit.com).
 
+## Tmux
+When working on our robots, you will often have the need to load up the same
+programs over and over. To make this process more convenient, we use tmux.
+Tmux is "a terminal multiplexer. It lets you switch easily between several
+programs in one terminal, detach them (they keep running in the background)
+and reattach them to a different terminal." To learn more about Tmux in general,
+check out the [Tmux Wiki](https://github.com/tmux/tmux/wiki). A quick reference
+sheet can be found [here](https://tmuxcheatsheet.com/).
+
+### Tmuxinator
+Managing Tmux sessions manually can be exteremly time consuming, so we use
+tmuxiator to provide a version controlled interface for our robots and
+simulators. Basic commands are:
+
+```sh
+$ tx list # show configs available to start
+
+$ tx start <config> # start (or join if already started) a tmuxinator session
+
+$ tmux ls # Show active all tmux sessions ( not only ones started by tmuxinator )
+
+$ tx stop <config> # stop a session that is running
+```
+
+The MIL configs for tmuxinator are hard linked from
+`$MIL_REPO/.tmuxinatorConfigs/<name>.yaml` to `~/.config/tmuxinator/<name>.yaml`.
+If you want, you can add personal tmuxinator configs you should add them in `.config`.
+If you need to add MIL-wide configs, place them in `.tmuxinatorConfigs`
+
+
+
```

Comment:
I think this should go below Vim/Emacs in this file, and possibly below bash/zsh. Only some members use tmux regularly.